### DISCUSSION TITLE
Need some help with deployment

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#4489] [Troubleshooting] [answered]
I set myself up with a Digital Ocean and Forge account, went through the steps for deployment as per the docs. The site works, ish... Everything loads except some parts of the header and footer, and some css styling is all screwed up. Also I get a "This site can’t be reached" when I first visit the site and if I reload it then the site loads.

For the deploy script I removed what was there and used the example in the docs:
```
cd /home/forge/{example}.{tld}
git pull origin main
npm install && npm run production
php please cache:clear
```

**Comment by fredbouin** [comment]
I found the issue with the css, tailwind wasn't purging my partials.

I still get the "site can't be reached" problem on first visit, it seems to me like a cache thing but clearing cache doesn't do anything.

**Reply by duncanmcclean** [reply]
How big is your site?

**Reply by fredbouin** [reply]
897mb

**Reply by duncanmcclean** [reply]
How many entries is that? 

If you're unsure, go into your terminal and run `php artisan tinker`, then run this code which will give you a total: `return \Statamic\Facades\Entry::all()->count();`

My theory is that if you're sites pretty big, it may be trying to build the Stache on the first page load and timing out because it's taking longer than 30 secs to do so.

**Reply by fredbouin** [reply]
That is only 18 entries right now, will have a lot more. 
This reminds me though, there is a really uselessly large image on the home page that I should remove. If your theory is right maybe that's the source of the problem. 

I'm struggling with a few things right now deployment wise and having some issues on the forge end of things, so I'll try the image theory after I resolve the forge problem!

**Reply by duncanmcclean** [reply]
That won't be it then: 18 entries isn't very many. I'd wouldn't expect that to happen until you have at least 100 entries. 🤔

**Comment by fredbouin** [comment, marked as answer]
Oh strange, I thought I had tested this already, apparently this is only an
issue in chrome on my machine. So not a big problem in the end but still
annoying, I would assume it's a cache issue, I cleared browsing data in
chrome and cleared cache, but it hasn't changed anything.

EDIT

answer: chrome is redirecting to https
